ibm_cloud_sdk_core/token_managers/container_token_manager.py

```diff
@@ -177,3 +178,15 @@ def set_iam_profile_id(self, iam_profile_id: str) -> None:
             iam_profile_id: id of the linked trusted IAM profile to be used when obtaining the IAM access token
         """
         self.iam_profile_id = iam_profile_id
+
+    def read_file(self, filename: str) -> str:
+        """Read in the specified file and return the contents as a string"""
```

Comment:
It seems the linter doesn't complain about missing docstrings, but I have a suggestion. It's up to you to change it or not. :)
```suggestion
        """Read in the specified file and return the contents as a string.

        Args:
            filename: ID of the linked trusted IAM profile to be used when obtaining the IAM access token.

        Returns:
            A string with the contents of the file.

        Raises:
            Exception: An error occured reading the file.
        """
```

Reply:
Good idea, but I'll use a different description for the filename arg :joy:
Sorry, couldn't resist :)

Reply:
fixed

Reply:
LOL, you got me! I was too lazy and copied the docstring from another method. 😂